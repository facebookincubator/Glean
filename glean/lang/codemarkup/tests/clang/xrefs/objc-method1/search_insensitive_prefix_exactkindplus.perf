{
  "@generated": null,
  "cxx1.DeclByName.5": 6,
  "cxx1.DeclarationNameSpan.5": 8,
  "cxx1.ObjcContainerDeclaration.5": 2,
  "cxx1.ObjcContainerDefinition.5": 2,
  "cxx1.ObjcMethodDeclaration.5": 2,
  "cxx1.ObjcMethodDeclarationName.5": 2,
  "cxx1.QName.5": 4,
  "cxx1.VariableDeclaration.5": 4
}