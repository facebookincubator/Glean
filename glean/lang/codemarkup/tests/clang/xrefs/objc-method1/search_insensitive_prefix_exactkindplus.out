[
  "@generated",
  {
    "key": {
      "searchcase": 1,
      "name": "foobar",
      "entity": {
        "cxx": {
          "decl": {
            "objcMethod": {
              "key": {
                "selector": { "key": [ "fooBar" ] },
                "locations": [
                  {
                    "file": { "key": "test.mm" },
                    "span": { "start": 440, "length": 6 }
                  }
                ],
                "container": { "implementation": { "key": "Bar" } },
                "signature": {
                  "key": { "returns": { "key": "void" }, "parameters": [ ] }
                },
                "isInstance": true,
                "isOptional": false,
                "isAccessor": false,
                "source": {
                  "file": { "key": "test.mm" },
                  "lineBegin": 30,
                  "columnBegin": 1,
                  "lineEnd": 33,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "location": {
        "name": "fooBar",
        "file": { "key": "test.mm" },
        "location": {
          "range": {
            "file": { "key": "test.mm" },
            "lineBegin": 30,
            "columnBegin": 1,
            "lineEnd": 33,
            "columnEnd": 1
          }
        },
        "destination": {
          "file": { "key": "test.mm" },
          "span": { "start": 440, "length": 6 }
        }
      },
      "kind": 7,
      "language": 0
    }
  }
]